### ISSUE TITLE
Error: Too many failures; giving up

### ISSUE THREAD

**Issue by ronnocol** [keybase/client#8769]
I had typed my passphrase into the OS X UI incorrectly a number of times, but now I remember it (verified that it works on the website.) But I can't login to my Mac any more... no matter what I do (even when I type the correct password) I just get "Error: Too many failures; giving up."

How do I get back into Keybase on my MBP w/o "recovering" my passphrase thus destroying my PGP keys and freezing my account for 5 days?

**Comment by maxtaco** [comment]
Can you log onto the website?

**Comment by ronnocol** [comment]
Yes. And my phone and Linux box are fine too. I just can’t get back into the UI on my OSX client.

**Comment by maxtaco** [comment]
Can you run `keybase log send` on the failing OSX machine?  You might need to "reprovision" that device, you might have hid an old bug that affects people's ability to login

cc: @patrickxb

**Comment by maxtaco** [comment]
Hmmm, no your account is too new.  If it's a bug, it's a new bug....

**Comment by patrickxb** [comment]
@ronnocol I would also try doing a full quit of the app.  

In the menubar widget, select the dropdown on the top right and select Quit.
Then launch Keybase again and try again.

Sorry for the bug, but that restart might allow you to login.

**Comment by ronnocol** [comment]
Sorry for the delay, problem was on my home laptop, and it's been a long day.

I've sent the logs as per the above:

------------
Success! Your log ID is:

  98e8d9a01b0f47c5a8816d1c

Here's a URL to submit new bug reports containing this ID:

  https://github.com/keybase/client/issues/new?body=[write%20something%20useful%20and%20descriptive%20here]%0A%0Amy%20log%20id:%2098e8d9a01b0f47c5a8816d1c

Thanks!
------------

Full quit and restart does not solve the problem.

I'm happy to leave this laptop in this state for the time being in case there's any additional information that you need, things you want to try, etc.

lgoc
